behat/features/bootstrap/FeatureContext.php

```diff
@@ -67,4 +67,133 @@ public function iVisitNodePageOfType($title, $type) {
   public function iShouldHaveAccessToThePage() {
     $this->assertSession()->statusCodeEquals('200');
   }
+
+  /**
+   * @Then I should not have access to the page
+   */
+  public function iShouldNotHaveAccessToThePage() {
+    $this->assertSession()->statusCodeEquals('403');
+  }
+
+  /**
+   * @When I start editing :title node of type :type
+   */
+  public function iStartEditingNodeOfType($title, $type) {
```

Comment:
Why remove this function? I'm using this step to check user can edit the repository.